scripts/actions/deserialize-html.js

```diff
@@ -74,7 +74,9 @@ const stripTranslateFrontmatter = () => {
       const frontmatterObj = yaml.load(tree.children[0].value);
       delete frontmatterObj.translate;
       delete frontmatterObj.redirects;
-      const frontmatterStr = yaml.dump(frontmatterObj).trim();
+      const frontmatterStr = yaml
+        .dump(frontmatterObj, { lineWidth: -1 })
```

Comment:
By changing this to -1 are we now not enforcing any line width at all? Before it was 80 characters, but now it's no limit? I'm just wondering if we should still enforce a line width here.

Reply:
it had no limit before, this is caused by the yaml parsing library we're using here which was introduced recently. i don't think there's any technical reason to enforce line width

Reply:
Ah ok, thanks for the explaination @LizBaker